### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by LadyAuroralite**

### CODE REVIEW

File: app/src/main/java/com/instructure/template/projectCodeHere/ProfilePage.java [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                List<GetEnrollments.EnrollmentResponse> g;
+                g = (ArrayList<GetEnrollments.EnrollmentResponse>)response.body();
+                int count=0;
+                double total=0;
+                for(GetEnrollments.EnrollmentResponse i : g) {
+                    total+=i.getGrades().getCurrent_score();
+                    count++;
+                }
+                double totalGPA=total/count,GPA=0;
+                if(totalGPA>90) {
```
Comment (HeyBanditoz) [inline app/src/main/java/com/instructure/template/projectCodeHere/ProfilePage.java]: This sucks. Can't you implement something equivalent to `(score_in_percentage / 20) - 1` instead? What if someone has a 130% on something?

File: app/src/main/java/com/instructure/template/projectCodeHere/ProfilePage.java [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.profile_page, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        logoutButton = view.findViewById(R.id.logoutButton);
+        textView = view.findViewById(R.id.textView);
```
Comment (HeyBanditoz) [inline app/src/main/java/com/instructure/template/projectCodeHere/ProfilePage.java]: Make these variable names a little more descriptive. Right click where you're defining it and `Refactor -> Rename...`

**Review by HeyBanditoz — CHANGES_REQUESTED** [review]
You know, we should probably make this logout code an object or something... I copy pasted it in my work too. Also squash your commits.

**Review by HeyBanditoz — APPROVED** [review]
¯\\\_(ツ)_/¯

**Review by LeaHiHi — APPROVED** [review]

**Pull request merged**